backends/p4tools/modules/testgen/targets/bmv2/bmv2.cpp

```diff
@@ -79,17 +79,18 @@ CompilerResultOrError Bmv2V1ModelCompilerTarget::runCompilerImpl(
         return std::nullopt;
     }
 
-    // Vector containing pairs of restrictions and nodes to which these restrictions apply.
-    P4ConstraintsVector p4ConstraintsRestrictions;
-    // Defines all "entry_restriction" and then converts restrictions from string to IR
-    // expressions, and stores them in p4ConstraintsRestrictions to move targetConstraints further.
-    program->apply(AssertsParser(p4ConstraintsRestrictions));
+    // Parses any @refers_to annotations and converts them into a vector of restrictions.
```

Comment:
FYI, we also introduced @referenced_by recently, which is equivalent except that the role of src and dst are swapped:
```
table src_table {
  ...
  @refers_to(dst_table, dst_field) src_field;
  ...
}
```
is the same as
```
table dst_table {
  ...
  @refers_to(dst_table, dst_field) dst_field;
  ...
}
```
I assume this is not currently supported here yet?